### ISSUE TITLE
form.is_valid() unnecessarily called twice

### ISSUE THREAD

**Issue by nerdoc** [django-commons/django-unicorn#375]
https://github.com/adamghill/django-unicorn/blob/c80ebd706d4d80d50fd1b0b660cf652845e5f17f/django_unicorn/components/unicorn_view.py#L430

`form.is_valid()` is already called in `self._get_form()` - this line could be removed.

**Comment by adamghill** [comment]
Good call. I'll clean that up when I add some additional form functionality in the next release.

**Comment by JohananOppongAmoateng** [comment]
@adamghill  i think we can close this because no longer seeing it being called twice in the code. I think your previous refactors might have taken care of it
